overlord/ifacestate/ifacemgr.go

```diff
@@ -71,12 +73,21 @@ func Connect(s *state.State, plugSnap, plugName, slotSnap, slotName string) (*st
 	// TODO: Store the intent-to-connect in the state so that we automatically
 	// try to reconnect on reboot (reconnection can fail or can connect with
 	// different parameters so we cannot store the actual connection details).
-	summary := fmt.Sprintf(i18n.G("Connect %s:%s to %s:%s"),
+	summary := fmt.Sprintf(i18n.G("Collect attributes of plug %s:%s"), plugSnap, plugName)
+	collectPlugAttr := hookstate.HookTask(s, summary, plugSnap, snap.Revision{N: 0}, "collect-plug-attr-"+plugName)
```

Comment:
`0` is the default value for an int, so you don't need to include it when creating the revision: `snap.Revision{}`